genotype/store/trending.py

```diff
@@ -0,0 +1,62 @@
+"""Bla bla. What do you want here???"""
+from genotype.store.models import Analysis, Genotype, Sample
+
+
+def build_snp_dict(analysis_id):
+    """Building a dict of snps for a specific analysis."""
+
+    snp_dict = {}
+    try:
+        genotypes = Genotype.query.filter(Genotype.analysis_id == analysis_id).all()
+    except:
+        genotypes = []
+    for genotype in genotypes:
+        snp_dict[genotype.rsnumber] = [genotype.allele_1, genotype.allele_2]
+
+    return snp_dict
+
+
+def compare(analysis_1, analysis_2):
```

Comment:
could you please hint at the return types

Reply:
👍